### PR TITLE
Contact page - Style

### PR DESCRIPTION
**Pull request by SolivanLau**
### Style
- added `display: flex` to main, centering all content regardless of `min-height: 90vh`
- added min height to form response. prevents height changing from DOM manipulation

### CODE REVIEW

File: css/contact.css [hunk 1]
```diff
@@ -21,8 +21,13 @@ header {
 main {
 	background: var(--grad-blue-purple);
 	color: var(--text-color-light);
-	padding: var(--spacing-xxl) 0;
+	padding: var(--spacing-xl) 0;
```
Comment (SolivanLau) [inline css/contact.css:24]: made padding between header and footer smaller to accommodate flex centering

File: css/contact.css [hunk 2]
```diff
@@ -21,8 +21,13 @@ header {
...
 	min-height: 90vh;
+
+	/* centering form in main */
+	display: flex;
+	justify-content: center;
+	align-items: center;
```
Comment (SolivanLau) [inline css/contact.css:30]: centering content in main

File: css/contact.css [hunk 3]
```diff
@@ -60,6 +65,12 @@ form .form-container {
 textarea[name="message-input"] {
 	min-height: 140px;
 }
+
+/* form response */
+.form-response-container {
+	/* persistent space for user response */
+	min-height: var(--spacing-med);
+}
```
Comment (SolivanLau) [inline css/contact.css:73]: persistent height for form response

**Review by maxharrisnet — APPROVED** [review]
This looks better, thanks! 🎈

**Pull request merged**